src/components/common/Text/Text.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import './Text.css'
+
+const Text = ({ type, text }) => {
+    if (type === 'text') {
+        return <input
```

Comment:
Line 10-15: Looking it over and doing a research this past week. I seemed to not have figured out what I'm doing wrong here. I know it should be an input but am I creating it correctly? I now this function should be very similar to the Flash.js function but what I question is that if we have multiple inputs how can we decipher what input is what?